_ml-commons-plugin/cluster-settings.md

```diff
@@ -73,7 +102,7 @@ plugins.ml_commons.monitoring_request_count: 100
 ### Value range
 
 - Default value: `100`
-- Value range: [0, 100,000,000]
+- Value range: [0, 10,000,000]
```

Comment:
Does the system interpret commas in numbers values? In other words, if you added this as a value, the system could read the number 10,000,000? 
If yes, you might add a comma to 10,000 in line 73.

Reply:
Fixed the three instances where we didn't use commas previously.